Makefile

```diff
@@ -1,3 +1,4 @@
 update_metadata:
-	(cd libPhoneNumberTests && ./metadataGenerator) && \
-		(cd libPhoneNumber && ./GeneratePhoneNumberHeader.sh)
+	cd libPhoneNumberTests && ./metadataGenerator
```

Comment:
Should this be given its own script instead? The Makefile is fine too but maybe we want to conditionalize this step based on a change to input json?

Reply:
The existing script always fetches the latest data, so my reading of this was that it was just a place to write a script. If we want to replace the Makefile with a script that's fine with me too, but I don't think there's any point in trying to make dependency-sensitive steps. (The conversion is pretty fast too.)

Reply:
I'm sorry, I misunderstood a couple things here:
1. This is a Pod, so I'm guessing we'd only really be running this Makefile when updating?
2. The existing behavior is to fetch data into the test directory. So this seems fine. Just threw me for a second.